### PR TITLE
split pickle from encryption

### PR DESCRIPTION
**Pull request by mmerickel**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -124,7 +129,7 @@
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,
 # so a file named "default.css" will overwrite the builtin "default.css".
-html_static_path = ['_static']
+#html_static_path = ['_static']
```
Comment (tseaver) [inline docs/conf.py]: Why?  Does `tox -e docs` barf for you because of git's absurd ignore-empty-directories bug^Wdesign choice?

Comment (mmerickel) [inline docs/conf.py]: Yes, that's why. FWIW I'm currently updating the tox.ini to support the coverage support in the other Pylons repos (100% on py2 + py3). That'll be a separate PR.

File: pyramid_nacl_session/tests/test_serializer.py [hunk 2]
```diff
@@ -20,7 +20,7 @@ def test_dumps(self):
         SECRET = 'SEEKRIT!' * 4  # 32 bytes
         NONCE = b'\x01' * 24
         APPSTRUCT = {'foo': 'bar'}
-        PICKLED = pickle.dumps(APPSTRUCT)
+        PICKLED = pickle.dumps(APPSTRUCT, pickle.HIGHEST_PROTOCOL)
```
Comment (tseaver) [inline pyramid_nacl_session/tests/test_serializer.py:23]: Ugh!  I forgot Python2 uses 0 as the default for BBB.

File: docs/conf.py (hunk 1 above)
Comment (tseaver) [inline docs/conf.py]: I think I'd rather have an empty `_static/placeholder.txt`, to avoid future headscratching when adding (hypothetically) an image to illustrate something.

Comment (mmerickel) [inline docs/conf.py]: Done.

**Comment by tseaver** [comment]
Does this close #2?
